packages/block-library/src/gallery/deprecated.js

```diff
@@ -80,6 +80,26 @@ export function getHrefAndDestination( image, destination ) {
 	return {};
 }
 
+/**
+ * Gets an Image block from gallery image data
+ *
+ * Used to migrate Galleries to nested Image InnerBlocks.
+ *
+ * @param  {Object} image    Image properties.
+ * @param  {string} sizeSlug Gallery sizeSlug attribute.
+ * @param  {string} linkTo   Gallery linkTo attribute.
+ * @return {Object}          Image block.
+ */
+export function getImageBlock( image, sizeSlug, linkTo ) {
+	return createBlock( 'core/image', {
+		...( image.id && image.id !== null && { id: parseInt( image.id ) } ),
```

Comment:
```suggestion
		...( image.id && { id: parseInt( image.id, 10 ) } ),
```

No need to check `image.id !== null` after already checking the id is truthy. Also, while changing the line is it worth adding the radix to the `parseInt` call?

Reply:
🤦 Doh! good catch, fixed, left the radix off though as the default is 10 so isn't really needed unless it is something other than that IMO